types/rimiti__stimmy/index.d.ts

```diff
@@ -0,0 +1,10 @@
+// Type definitions for @rimiti/stimmy 1.8
+// Project: https://github.com/rimiti/stimmy
+// Definitions by: Dimitri DO BAIRRO <https://github.com/rimiti>
+// Definitions: https://github.com/DefinitelyTyped/DefinitelyTyped
+
+declare function replacer(start?: string, end?: string): (val: string, replace: {} | string[] | number[]) => string;
+
+declare namespace replacer {}
```

Comment:
This is an inappropriate hack. [Please read more here from the DefinitelyTyped README](https://github.com/DefinitelyTyped/DefinitelyTyped/tree/fbba9f55beab742b9d1c0fa9ef714e0771ad9d05#should-i-add-an-empty-namespace-to-a-package-that-doesnt-export-a-module-to-use-es6-style-imports)

Reply:
I'm agree with you but the package export a global function. 
I can not do otherwise. It would be necessary to correct the package...